### PR TITLE
[2|CBH - Report Dashboard] Report Tables

### PR DESCRIPTION
**Pull request by thejasveer**
Report Dashboard Ui 80% done

### CODE REVIEW

**Comment by Simpcyclassy** [comment]
Could you add screenshots of what you have done @thejasveer

**Comment by Ryan-L3** [comment]
@Simpcyclassy Can confirm that it's working

**Comment by Simpcyclassy** [comment]
The link leads me back to this PR

**Comment by thejasveer** [comment]
Screenshot 2024-10-06 at 2 35 14 PM

File: .gitignore [hunk 1]
```diff
@@ -45,4 +45,7 @@ yarn-error.log*
 stacks
 
 # JetBrains
-.idea
\ No newline at end of file
+.idea
+
+.lh
+.Ih
```
Comment (Simpcyclassy) [inline .gitignore:51]: Can you add new line?

File: .prettierignore [hunk 2]
```diff
@@ -37,3 +37,4 @@ next-env.d.ts
 
 pnpm-lock.yaml
 
+*.json
```
Comment (Simpcyclassy) [inline .prettierignore:40]: Here as well

File: package.json [hunk 3]
```diff
@@ -41,6 +42,7 @@
     "@supabase/supabase-js": "^2.39.3",
     "@tanstack/react-query": "^5.17.15",
     "@tanstack/react-query-devtools": "^5.17.15",
+    "@tanstack/react-table": "^8.20.5",
```
Comment (Simpcyclassy) [inline package.json:45]: Where did you use this?

Comment (thejasveer) [reply package.json:45]: @Simpcyclassy so basically shadcn need to integrate this to work

File: src/components/Report/Report.tsx [hunk 4]
```diff
@@ -0,0 +1,472 @@
...
+        className="absolute right-0 top-0"
+        onClick={() => {
+          /* Optional: Handle search button click */
+        }}
+      >
+        <Search size={15} />
+      </Button>
+    </div>
+  )
+}
```
Comment (Simpcyclassy) [inline src/components/Report/Report.tsx:472]: There is a lot going on in this file. Please can you put the child components into different files

Comment (thejasveer) [reply src/components/Report/Report.tsx:472]: yes

Comment (thejasveer) [reply src/components/Report/Report.tsx:472]: will make new pull request

**Comment by Simpcyclassy** [comment]
The rest looks good to me

File: src/components/Report/Report.tsx (hunk 4 above)
Comment (Simpcyclassy) [reply src/components/Report/Report.tsx:472]: With this same issue or another issue?

Comment (thejasveer) [reply src/components/Report/Report.tsx:472]: same

**Comment by Simpcyclassy** [comment]
Closing this for https://github.com/Builder-s-League/BuildersLeague-Edition1/pull/198

**Pull request closed without merging**